From 1d64c6975a3c4328b1bdac51de4c74c3efef3b31 Mon Sep 17 00:00:00 2001
Message-Id: <1d64c6975a3c4328b1bdac51de4c74c3efef3b31.1421709247.git.chang-joon.lee@intel.com>
In-Reply-To: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
References: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
From: Shobhit Kumar <shobhit.kumar@intel.com>
Date: Thu, 4 Sep 2014 18:34:26 +0530
Subject: [PATCH 19/75] REVERTME [VPG]: drm/i915: Update VBT MIPI config block
 definition for DSI Burst mode.

Upstream following patch has the code but not yet pulled in -

	commit 7f0c860533ff2de4b3bb84f71d5ce238fffe4d63
	Author: Shobhit Kumar <shobhit.kumar@intel.com>
	Date:   Wed Jul 30 20:34:57 2014 +0530

	drm/i915: Add support for Video Burst Mode for MIPI DSI

For: GMINL-3547
Change-Id: Id5664286d1f2ea9762b423b2fa17c78025ba9372
Signed-off-by: Shobhit Kumar <shobhit.kumar@intel.com>
---
 drivers/gpu/drm/i915/intel_bios.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/i915/intel_bios.h b/drivers/gpu/drm/i915/intel_bios.h
index 7241db7..25b7532 100644
--- a/drivers/gpu/drm/i915/intel_bios.h
+++ b/drivers/gpu/drm/i915/intel_bios.h
@@ -811,7 +811,8 @@ struct mipi_config {
 
 	u16 rsvd4;
 
-	u8 rsvd5[5];
+	u8 rsvd5;
+	u32 target_burst_mode_freq;
 	u32 dsi_ddr_clk;
 	u32 bridge_ref_clk;
 
-- 
1.7.9.5

